### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Romain Humair and Lucas Lattion

### DISCUSSION THREAD

**Discussion by lucaslattion** [heig-vd-dai-course/heig-vd-dai-course#86] [Show and tell]
The CLI will take an input file and an output file as arguments. It will also take the input file encoding and the output file encoding as optional arguments (the default will be UTF-8). **The CLI will search an argument text in the input file and will write the number of find element in the output file.**  It will display a message on success and a message on failure.

**Comment by ludelafo** [comment]
Looks good to me!

I discussed about quite the same idea with some of the groups to create a dictionary of all the words in the text with the number of times the word(s) appeared in the give file, was it with you? I'm sorry, I sometimes mix names and faces.

If it was with you, I would try to have a first working version for a given word and then do the same for all the words in the input file. By default, it could do it for all the words but you could specify with an optional argument (`--word test` for example) to juste do it for the word `test`.

**Reply by lucaslattion** [reply]
Link to the repository : https://github.com/lucaslattion/DAI-2023-24-Practical-work-1-CLI-Romain-Humair-and-Lucas-Lattion

**Comment by lucaslattion** [comment]
Last commit hash: a88e1d27c1449e1d387e048a53477e1d9d6d5f5f
Link: https://github.com/lucaslattion/DAI-2023-24-Practical-work-1-CLI-Romain-Humair-and-Lucas-Lattion/commit/a88e1d27c1449e1d387e048a53477e1d9d6d5f5f
